release.toml

```diff
@@ -0,0 +1,4 @@
+enable-all-features = true
+pre-release-hook = [
+  "../../gen-changelog.sh",
```

Comment:
Is this a list of commend? Can't we put the 2 commands of the script instead of having another file?